pxr/base/tf/stringUtils.cpp

```diff
@@ -869,7 +869,7 @@ TfDictionaryLessThan::_LessImpl(const string& lstr, const string& rstr) const
                 curEnd = lcur + std::min(std::distance(lcur, lend),
                                          std::distance(rcur, rend));
             }
-            else if (IsDigit(l) | IsDigit(r)) {
```

Comment:
Same as above.

Reply:
I've backed out those changes, just leaving the ones in usdSkel